### PR TITLE
Fixed the edit profile normalization. Fixed the comment count discrep…

### PR DESCRIPTION
**Pull request by chrislombaard**
…ancy, fixed image styling on the FED side, fixed error handling on the edit profile page, fixed topic of the day listing heading and article pagination, fixed comments date time template tag, fixed comments so that you can add a comment anonymously.

### CODE REVIEW

File: nurseconnect/forms.py [hunk 1]
```diff
@@ -198,13 +189,20 @@ def clean_username(self):
         # TODO: this check doesn't take into account the normalization
```
Comment (phala) [inline nurseconnect/forms.py]: Remove this comment

File: nurseconnect/templates/core/tags/topic_of_the_day.html [hunk 2]
```diff
@@ -4,15 +4,15 @@
 {% if articles %}
     <section class="ContentList">
         <header class="ContentList-header">
-            <h1 class="ContentList-categoryLabel">{% trans "Topic of the day" %}</h1>
+            <h1 class="ContentList-categoryLabel">{% trans "Latest Articles" %}</h1>
```
Comment (phala) [inline nurseconnect/templates/core/tags/topic_of_the_day.html]: I thought they requested that we remove the heading. Did they mean it should be under "Latest Articles"?
Also, this section will only be rendered if there is a Topic of the day article. I think the heading could possibly not be rendered at all if it is here

**Review by phala — CHANGES_REQUESTED** [review]

File: nurseconnect/templates/core/tags/topic_of_the_day.html (hunk 2 above)
Comment (chrislombaard) [reply nurseconnect/templates/core/tags/topic_of_the_day.html]: Ok, I'll move it to outside the topic of the day template. Maybe in the main.html file?

**Review by phala — COMMENTED** [review]
@chrislombaard do they want the Topic of the Day to appear under the heading "Latest articles"

**Review by phala — APPROVED** [review]

**Pull request merged**